cartridges/importer/importer.py

```diff
@@ -69,6 +70,8 @@ def __init__(self) -> None:
         self.game_pipelines = set()
         self.sources = set()
 
+    """PUBLIC"""
```

Comment:
How would you do it then? Or you want me to remove them?

I added them because it was hard for me to track what code could be touched in a thread. Otherwise you need to rely on name or following the code again and again.

Reply:
You can add comments to those (though I wouldn't do all uppercase), but private/public is indicated by the function name itself so I'd remove those.

Reply:
Sure